### DISCUSSION TITLE
How to keep the training results to two decimal places？

### DISCUSSION THREAD

**Discussion by jiaranjintianchism** [espnet/espnet#5131] [Q&A] [answered]
I want to keep two decimal places in the training results. Does anyone know what to do?

**Comment by tjysdsg** [comment, marked as answer]
If you are referring to the ASR results, you can modify sctk's source code under `tools/sctk` as described [here](https://groups.google.com/g/kaldi-help/c/5M6eTOT7UaQ). Then run `make config; make all; make install` for the changes to take effect.
